config.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+func get_config() metadata_S3Config {
+	parseConfig()
+	s3_conf := configS3Storage()
+	return s3_conf
+
+}
+
+type metadata_S3Config struct {
```

Comment:
I can't comment about functinalIty just for now :) but this metadata_S3Config struct has field names in camel case. Is there a reson for not using CamelCase for everything. For example, Web_metadata_folder can be renamed to WebMetadataFolder. Same for all other namings in the code, i think i am used to see that CamelCase is preferred.

Reply:
Thank you, changed now